### PR TITLE
Jar fix

### PR DESCRIPTION
**Pull request by Verzijl**
The program is now able to run from the jar. The jar to use is "jar with dependencies", this is the complete stand-alone.

<bug

### CODE REVIEW

**Comment by Verzijl** [comment]
Fixed #215.

**Comment by Verzijl** [comment]
Still needs some checkstyle fixes.

File: pom.xml [hunk 1]
```diff
@@ -94,20 +94,48 @@
...
+<!--                        </manifest>-->
+<!--                    </archive>-->
+<!--                </configuration>-->
+<!--            </plugin>-->
+
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>2.4</version>
+				<groupId>org.apache.maven.plugins</groupId>
```
Comment (toinehartman) [inline pom.xml]: Fix tab indentation, file has space indentation.

< layout

**Comment by JPluim** [comment]
Checkstyle gives 2 errors (GameEndController line 45 > 100 characters, Level method readMap too many lines). The rest works good on my Mac (can't test Linux), so when checkstyle is fixed :shipit:
<layout

**Pull request merged**